libs/counterfactuals/src/util/largeCounterfactualsView/getCounterfactualsScatter.ts

```diff
@@ -0,0 +1,224 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  FluentUIStyles,
+  IGenericChartProps,
+  JointDataset
+} from "@responsible-ai/core-ui";
+import { WhatIfConstants } from "@responsible-ai/interpret";
+import { localization } from "@responsible-ai/localization";
+import { PointMarkerOptionsObject } from "highcharts";
+import { Dictionary } from "lodash";
+
+import { buildScatterTemplate } from "./buildScatterTemplate";
+
+export interface IDatasetExplorerSeries {
+  name?: string;
+  color: any;
+  data: IDatasetExplorerData[];
+  marker?: PointMarkerOptionsObject;
+}
+export interface IDatasetExplorerData {
```

Comment:
This name seems a bit ambiguous. `DatasetExplorer` is a component, so this indicates to me that it's the data for it. Perhaps something with "data point" rather than just "data" would help?

Reply:
Thank you, removed unused interface.